README.md

```diff
@@ -83,15 +62,15 @@ Note that we will not be implementing the entire app in this quickstart, so the
 
 * *Your `$KUBECONFIG` should specify a valid `kubeconfig` file*, which points at the cluster you want to use for this demonstration.
 
-### Script
+### Example flow
 
-Copy and paste the script below to deploy the container image for a basic web app UI:
+You can copy and paste the commands below to deploy the web app UI:
 
 ```bash
 # Start by creating your app directory
```

Comment:
It might be good to make note here that the directory will be created at your current path.